commands/last.js

```diff
@@ -0,0 +1,18 @@
+const nbaUtil = require('../util/nbaUtils')
+const yUtils = require('../util/yahooUtils')
+
+module.exports = {
+  name: 'last',
```

Comment:
should we rename this command to prev or prev week?

Reply:
I vote for 'previous'

Reply:
Changed to previous